go/vt/vtgate/plan_execute.go

```diff
@@ -34,6 +36,25 @@ import (
 type planExec func(ctx context.Context, plan *engine.Plan, vc *vcursorImpl, bindVars map[string]*querypb.BindVariable, startTime time.Time) error
 type txResult func(sqlparser.StatementType, *sqltypes.Result) error
 
+func waitForNewerVSchema(ctx context.Context, e *Executor, lastVSchemaCreated time.Time) bool {
+	timeout := 5 * time.Second
```

Comment:
Seems a bit low, no? Is this used outside of tests?

Reply:
Increased to 30s. Note however that this wait only happens due to a small race, where the buffering has ended because that logic has seen the vschema change that indicates that SwitchWrites has completed. So we are not really waiting this long for SwitchWrites, just for the other SrvVSchema watcher subscriber thread to get the update.